auto_update_tests.py

```diff
@@ -72,6 +73,32 @@
 
       with open(expected_file, 'w') as o: o.write(actual)
 
+print '\n[ checking wasm-link-metadata testcases... ]\n'
+
+lld_path = os.path.join('test', 'lld')
+for obj_file in sorted(os.listdir(lld_path)):
```

Comment:
This changes the behavior slightly subtly because `os.listdir('test/foo/')` returns `['a', 'b', 'c']`, but `glob.glob('test/foo/*')` returns `['test/foo/a', 'test/foo/b', '/test/foo/c']`
This is actually even more convenient because we get paths directly and don't need to append `'test/foo'` everywhere.